projects/04_cube_xr/main.cpp

```diff
@@ -234,16 +245,53 @@ void ProjApp::Setup()
 
 void ProjApp::Render()
 {
-    PerFrame& frame = mPerFrame[0];
-
-    uint32_t currentViewIndex = 0;
+    PerFrame& frame            = mPerFrame[0];
+    uint32_t  imageIndex       = UINT32_MAX;
+    uint32_t  currentViewIndex = 0;
     if (GetSettings()->enableXR) {
         currentViewIndex = GetXrComponent().GetCurrentViewIndex();
     }
 
-    grfx::SwapchainPtr swapchain = GetSwapchain(currentViewIndex);
+    // render UI into a different composition layer
```

Comment:
nit: comment formatting (capitalization, end period). Same for other comments

Reply:
fixed